common/models/config-file.js

```diff
@@ -197,49 +196,48 @@ module.exports = function(ConfigFile) {
     }));
 
     async.map(patterns, find, function(err, paths) {
-      if(err) return cb(err);
+      if (err) return cb(err);
 
       // flatten paths into single list
       var merged = [];
       merged = merged.concat.apply(merged, paths);
 
       var configFiles = merged.map(function(filePath) {
-        return new Ctor({path: filePath});
+        return new Ctor({ path: filePath });
       });
       cb(null, configFiles);
     });
 
     function find(pattern, cb) {
       glob(pattern, { cwd: workspaceDir }, cb);
     }
-  }
+  };
 
   ConfigFile.prototype.getExtension = function() {
     return path.extname(this.path);
-  }
+  };
 
   ConfigFile.prototype.getDirName = function() {
     return path.basename(path.dirname(this.path));
-  }
+  };
 
   ConfigFile.prototype.getFacetName = function() {
     var dir = this.getDirName();
     // NOTE: glob always returns the path using forward-slash even on Windows
     // See: https://github.com/strongloop/generator-loopback/issues/12
     var baseDir = this.path.split('/')[0];
 
-    if(dir === ROOT_COMPONENT
-      || baseDir === this.path
-      || baseDir === 'models') {
+    if (dir === ROOT_COMPONENT || baseDir === this.path ||
+      baseDir === 'models') {
```

Comment:
```
var isRootComponent = dir === ROOT_COMPONENT
  || baseDir === this.path
  || baseDir === 'models';

var facetName = isRootComponent ? ROOT_COMPONENT : baseDir;

return facetName;
```

I prefer not having to parse the details to figure out what the intent of the function is (ie. you are trying to return a facet name, i don't need to see the guts `return isRootComponent ? ...` to determine this.